src/main/scala/cdevel.scala

```diff
@@ -2,7 +2,7 @@ package ohnosequencesBundles.statika
 
 import ohnosequences.statika._, bundles._, instructions._
 
-case object cdevel extends Bundle() {
+abstract class Cdevel extends Bundle() {
```

Comment:
then it's simpler in the other bundles: they just depend on `cdevel` (as it was before)